### PR TITLE
add summer answering times and midsommar day answering times

### PR DESCRIPTION
**Pull request by AlexSJ1990**
summer weekend answering times (from weekend Saturday 13th June to Sunday 9th August)
Saturdays and Sundays chat will open at 10am

Midsommar (19th June)
chat will be open from 10-6pm with answering times of an hour

### CODE REVIEW

File: src/main/java/com/hedvig/botService/chat/StatusBuilderImpl.java [hunk 1]
```diff
@@ -247,16 +206,12 @@ public String getStatusMessage(Clock c, Locale locale) {
       return getRepliesTomorrow(locale);
     }
 
-    if ((todayDate.equals(midsommarStartDate) && hour >= 23) || (todayDate.equals(midsommarEndDate))) {
-      return "Hedvig svarar den 22e juni"; //TODO lets fix support for dates later!
+    if (todayDate.equals(midsommar)) {
+      getMidsommarAnsweringTimes(hour, locale);
```
Comment (FredrikAreschoug) [inline src/main/java/com/hedvig/botService/chat/StatusBuilderImpl.java:210]:
```suggestion
      return getMidsommarAnsweringTimes(hour, locale);
```
or am I wrong?

**Comment by FredrikAreschoug** [comment]
And let us deploy this together I have some stuff in master :blush:

File: src/main/java/com/hedvig/botService/chat/StatusBuilderImpl.java (hunk 1 above)
Comment (meletios) [reply src/main/java/com/hedvig/botService/chat/StatusBuilderImpl.java:210]: GOOOOD CATCH!

Comment (meletios) [reply src/main/java/com/hedvig/botService/chat/StatusBuilderImpl.java:210]: GOOOD catch

**Pull request merged**